test/runtests.py

```diff
@@ -759,6 +755,10 @@ def main():
     print('[{} seconds] {}'.format(
         round(elapsed_time.total_seconds(),2), 'Success!' if not failed else 'Failed!'))
 
+    # rm profile.dpl.*
+    for f in glob.glob(test_root + '/*/profile.dpl.*'):
+        os.remove(f)
+
```

Comment:
Another ride-along - moved this down from line 744, delete dynaimc profile info after running tests rather than before, because I've had enough of it filling up my source tree.